juno/assetlist.json

```diff
@@ -251,6 +251,32 @@
         "logo_URIs": {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/joe.png"
         }
+      },
+      {
+        "description": "Governance Token for Digital Land Acquisition DAO",
+        "type_asset": "cw20",
+        "address": "juno1sfwye65qxcfsc837gu5qcprcz7w49gkv3wnat04764ld76hy3arqs779tr",
+        "denom_units": [
+          {
+            "denom": "cw20:juno1sfwye65qxcfsc837gu5qcprcz7w49gkv3wnat04764ld76hy3arqs779tr",
+            "exponent": 0
+          },
+          {
+            "denom": "dla",
+            "exponent": 6
+          }
+        ],
+        "base": "juno1sfwye65qxcfsc837gu5qcprcz7w49gkv3wnat04764ld76hy3arqs779tr",
+        "name": "Digital Land Acquisition DAO",
+        "display": "DLA DAO",
+        "symbol": "DLA",
+        "logo_URIs": {
+          "png": "https://github.com/frameworkfortune/chain-registry/blob/master/juno/images/dla.png?raw=true",
```

Comment:
can we please change the URI to: https://github.com/cosmos/chain-registry/blob/master/juno/images/dla.png

Reply:
Yes